### ISSUE TITLE
Use CORE_SYMBOL instead of hardcoded EOS

### ISSUE THREAD

**Issue by kesar** [DAppTalking/signupeoseos#1]
If you use CORE_SYMBOL instead of EOS, this script can work for any side chain / fork of EOS. 👍 

Example:

```
    eosio_assert(quantity.symbol == CORE_SYMBOL, "signupeoseos only accepts core tokens for signup account");
```

**Comment by thbourlove** [comment]
👍

**Comment by xJonathanLEI** [comment]
Haha I just submitted a PR for this, and then I found this issue.

**Comment by xJonathanLEI** [comment]
#2

**Comment by thbourlove** [comment]
Closed cause I merged #2.
